docs/docs-content/clusters/public-cloud/azure/required-permissions.md

```diff
@@ -46,254 +46,907 @@ You can use Validator with the Palette CLI. Check out the Palette CLI
+             "Microsoft.Network/privateDnsZones/virtualNetworkLinks/read",
+             "Microsoft.Network/privateDnsZones/virtualNetworkLinks/write",
+             "Microsoft.Network/privateDnsZones/write",
+             "Microsoft.Network/publicIPAddresses/delete",
+             "Microsoft.Network/publicIPAddresses/join/action",
+             "Microsoft.Network/publicIPAddresses/read",
+             "Microsoft.Network/publicIPAddresses/write",
+             "Microsoft.Network/routeTables/delete",
+             "Microsoft.Network/routeTables/read",
+             "Microsoft.Network/routeTables/write",
+             "Microsoft.Network/virtualNetworks/delete",
+             "Microsoft.Network/virtualNetworks/join/action",
+             "Microsoft.Network/virtualNetworks/join/action",
+             "Microsoft.Network/virtualNetworks/joinLoadBalancer/action",
+             "Microsoft.Network/virtualNetworks/peer/action",
+             "Microsoft.Network/virtualNetworks/read",
+             "Microsoft.Network/virtualNetworks/subnets/delete",
+             "Microsoft.Network/virtualNetworks/subnets/join/action",
+             "Microsoft.Network/virtualNetworks/subnets/joinLoadBalancer/action",
+             "Microsoft.Network/virtualNetworks/subnets/read",
+             "Microsoft.Network/virtualNetworks/subnets/virtualMachines/read",
+             "Microsoft.Network/virtualNetworks/subnets/write",
+             "Microsoft.Network/virtualNetworks/virtualMachines/read",
+             "Microsoft.Network/virtualNetworks/virtualNetworkPeerings/delete",
+             "Microsoft.Network/virtualNetworks/virtualNetworkPeerings/read",
+             "Microsoft.Network/virtualNetworks/virtualNetworkPeerings/write",
+             "Microsoft.Network/virtualNetworks/write",
+             "Microsoft.Resources/subscriptions/resourceGroups/read",
+             "Microsoft.Storage/storageAccounts/blobServices/containers/read",
+             "Microsoft.Storage/storageAccounts/blobServices/containers/write",
+             "Microsoft.Storage/storageAccounts/read",
+             "Microsoft.Storage/storageAccounts/write"
+           ],
+           "NotActions": [],
+           "AssignableScopes": ["/subscriptions/$SUBSCRIPTION_ID"]
+         }
+         ```
+
+       </details>
+
+3.  Create the custom role, _Dynamic Placement Palette Deployer_, in Azure using the JSON file you created in the
+    previous step. Issue the following command to create the role.
+
+    ```shell
+    az role definition create --role-definition @dynamic-permissions.json
+    ```
+
+4.  Export the
+    [security principal](https://learn.microsoft.com/en-us/azure/role-based-access-control/overview#security-principal)
+    object ID you want to use with Palette to a variable.
+
+    ```shell
+    export ASSIGNEE="<security_principal_object_id>"
+    ```
+
+5.  Assign the role by creating a role assignment referencing the role definition `Dynamic Placement Palette Deployer`.
```

Comment:
We used italics for the the role name before. Maybe we can stick to the same formatting - either both italics or both code font?

Reply:
Good catch :)